inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -54,7 +55,8 @@ def simulate(
             resource_pool_id: Optional[UUID] = None,
             simulation_time: float = 10,  # ns
             integrator: Literal["md", "sd", "bd"] = "md",
-            nsteps_minim: int = 5000):
+            nsteps_minim: int = 5000,
+            visualized_section: str = "Protein-H"):
         """Simulate the solvation of a protein.
 
         Args:
```

Comment:
Add the new parameter to the documentation and mention the options possible for this argument.